test/rubygems/test_gem_installer.rb

```diff
@@ -1142,14 +1142,24 @@ def test_install_extension_dir_is_removed_on_reinstall
   end
 
   def test_find_lib_file_after_install
```

Comment:
hence why I think we might want two tests -- one that continues to work for <= 1.9.2, and one that works for the latest versions